detox/src/utils/childProcess/exec.js

```diff
@@ -14,9 +14,10 @@ async function execWithRetriesAndLogs(bin, options = {}) {
     interval = 1000,
     prefix = null,
     args = null,
-    timeout = 0,
+    timeout,
```

Comment:
Zero is a default value, so there is no need to pass that explicitly.
This helps to make tests a bit cleaner.